### PR TITLE
added extra cvxopt installation step

### PR DESCRIPTION
**Pull request by aechevarria1**
Did I put this in the right place? Also is it fine to have Mac and Linux be separated?

### CODE REVIEW

**Comment by aechevarria1** [comment]
Same unit tests keep failing, with reference to #541

**Comment by galbramc** [comment]
test this please

**Comment by galbramc** [comment]
If you want to install cvxopt in your system python, you need sudo for the mac as well. The problem is if you install gpkit with sudo, then the ~/.gpkit belongs to root and you don't have permissions to use it. This means someone really can't install gpkit in the system python...

File: docs/source/installation.rst [hunk 1]
```diff
@@ -65,9 +65,13 @@ For Windows 10, cvxopt does not appear to be an option.
...
 -----------------
 
-Mac OSX and Linux
+Mac OSX
     Run ``pip install cvxopt``
-    
+
+Linux
+    Run ``sudo apt-get install libblas-dev liblapack-dev``
+    then ``sudo pip install cvxopt``
```
Comment (whoburg) [inline docs/source/installation.rst]: @aechevarria1, I recommend addressing @galbramc's point by removing `sudo ` from both lines above.

**Comment by whoburg** [comment]
Review complete. Once ready to merge, this PR will close #775.

**Comment by galbramc** [comment]
No you have to use sudo with apt-get, but the pip install is the question. I don't know how to take a stance on if you should recommend to users to install things in the system, or push them towards installing things in virtualenvs. That is something you have to decide. Regardless, installing gpkit in the system, as things currently are, does not actually work correctly because of the permission problems with ~/.gpkit.

**Comment by galbramc** [comment]
If you don't have a virtualenv, then we actually need a sudo command for OSX as well. I just don't know how to formulate recommending to users if they should install everything in their system, or if we should urge them to use virtualenvs.

**Comment by aechevarria1** [comment]
I looked through a lot of popular python packages and their installation processes, and it seems that the majority of them use `pip install`. I didn't notice any of the packages having instructions for use of virtualenvs

**Comment by bqpd** [comment]
Yeah, I think they generally assume that if you're installing in a virtualenv you'll know how to set it up yourself.

The permissions issue could be fixed just by chmod'ing `.gpkit`...

**Comment by aechevarria1** [comment]
So are we going with `pip install` for both mac and linux and `sudo apt-get ...` for linux only?

**Comment by whoburg** [comment]
This can be closed without merging once #934 is merged. Resolution to this was 9a5bb4d833e22a5ba775fba1b6487b25b488b098.

**Pull request closed without merging**